arcgis-ios-sdk-samples/Display information/Sketch on the map/SketchViewController.swift

```diff
@@ -48,6 +48,11 @@ class SketchViewController: UIViewController {
         self.map.initialViewpoint = AGSViewpoint(targetExtent: AGSEnvelope(xMin: -10049589.670344, yMin: 3480099.843772, xMax: -10010071.251113, yMax: 3512023.489701, spatialReference: .webMercator()))
     }
     
+    deinit {
+        //remove self as observer for notifications
+        NotificationCenter.default.removeObserver(self, name: .AGSSketchEditorGeometryDidChange, object: nil)
```

Comment:
Was this change due to a rule violation? If so, I would not be in favor of the rule. This (removing `self` as an observer in `deinit`) has not been necessary since like iOS 9.

Reply:
Oh, if the observation removal isn't necessary then yeah there isn't much point to the rule.